### PR TITLE
r-biobase: new package

### PR DESCRIPTION
**Pull request by baberlevi**
adding new package r-biobase with dependency r-biocgenerics

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-biocgenerics/package.py [hunk 1]
```diff
@@ -22,19 +22,13 @@
...
-
-    homepage = 'https://bioconductor.org/packages/BiocGenerics/'
-    version('3.3',
-            git='https://github.com/Bioconductor-mirror/BiocGenerics.git',
-            branch='release-3.3')
-    version('3.2',
-            git='https://github.com/Bioconductor-mirror/BiocGenerics.git',
-            branch='release-3.2')
-
-    depends_on('r@3.3.0:3.3.9', when='@3.3')
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/r-biocgenerics/package.py]: Does this no longer depend on R?

Comment (junghans) [reply var/spack/repos/builtin/packages/r-biocgenerics/package.py]: `RPackage` depend on `r` automatically.

Comment (junghans) [reply var/spack/repos/builtin/packages/r-biocgenerics/package.py]: `RPackage` depend on `r` automatically.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**